### ISSUE TITLE
Set up spam protection for the helpdesk

### ISSUE THREAD

**Issue by malemburg** [EuroPython/ep-helpdesk#6]
The system offers two options for spam protection:

https://django-helpdesk.readthedocs.org/en/latest/spam.html

We should set up one of them, since we're getting a bit of spam.

**Comment by malemburg** [comment]
The two options don't appear to work anymore.

A different solution would be to setup a proper mail server for EuroPython and use its spam protection tools instead.

**Comment by alexsavio** [comment]
https://github.com/tomav/docker-mailserver ?

**Comment by malemburg** [comment]
Yes, I've already looked at it. We should probably give this a spin.
